### ISSUE TITLE
Place 'Sign Up' and 'Log In' links so that they are visible without scrolling.

### ISSUE THREAD

**Issue by Golutazem** [misskey-dev/misskey#4900]
I do not know whether you wish this to be categorized as a 'Bug', because it is not a bug, but is a problem.

Currently, on my screen, the 'Sign Up' and 'Log In' links are not visible unless the user scrolls down in a content box.  This makes them difficult to find.  Change their placement so that they are visible without the user scrolling anything.  They could be placed at the top of the content section they are currently in or outside of that.

**Comment by AyaMorisawa** [comment]
Technically, it's more like a usability problem than a bug.

In my opinion, on the front page, the list in order of priority should be something like:

* log in
* sign up
* language setting (not shown in the current version)
* others

**Comment by Golutazem** [comment]
Language setting selection is not desirable if the application retrieves it from the user's browser.

**Comment by AyaMorisawa** [comment]
> Language setting selection is not desirable if the application retrieves it from the user's browser.

Some users prefer a different language than the browser's language. Besides, unfortunately, [Misskey's localization is incomplete for most languages](https://crowdin.com/project/misskey) at the moment. Therefore, language selection makes it more user-friendly.

**Comment by Golutazem** [comment]
If the user wishes to use a different language for Misskey than other WWW sites, they can run it in a different profile. Both Mozilla and Chromium derived browsers support multiple profiles for the same user.

What would be especially bad is if the user's selection is not used by default. If Misskey supports using the brower's setting as the default choice then having an additional possibility of selecting the language by the UI is not a problem for users.

**Comment by AyaMorisawa** [comment]
Using the browser's setting as the default looks good to me. I'm pretty sure that language selection is desirable because it's annoying to switch profiles only for changing the browser's language.

**Comment by KisaragiEffective** [comment]
![image](https://github.com/user-attachments/assets/4e818e69-2054-4b55-a3b0-7113eab932ab)

triage: this is resolved in 2023.5.0 or earlier version. Please let me known if the intention is not about links in rendered top-page when not logged-in.
